e2e/js_run_devserver/multirun_test.sh

```diff
@@ -14,107 +14,119 @@ _sedi () {
   sed "${sedi[@]}" "$@"
 }
 
-echo "TEST - $0: $1"
+echo "$$: TEST - $0: $1"
```

Comment:
I found putting the pid as a prefix helpful to distinguish between the stdout of the shell script, bazel and the http server.